src/APIClient/NativeApiClient.ts

```diff
@@ -6,6 +6,18 @@ import type {
     WithDefault,
 } from "react-native/Libraries/Types/CodegenTypes";
 
+type ClientResponseMetrics = {
+    networkType: string;
+    tlsCipherSuite: string;
+    tlsVersion: string;
+    httpVersion: string;
+    isCached: boolean;
+    compressedSize: Double;
+    size: Double;
+    connectionTime: Double;
+    latency: Double;
+};
```

Comment:
Shouldn't this be the same as APIClient.ts?  Can't we define here and re-export there?

Missing at least `speedInMbps`, maybe?

Reply:
fixed